lib/mobility.rb

```diff
@@ -95,6 +95,14 @@ def extended(model_class)
       def model_class.translates(*args, **options)
         include Mobility.translations_class.new(*args, **options)
       end
+
+      # FIXME move this elsewhere?
+      def model_class.has_translated_rich_text(name)
+        has_rich_text(name)
+        has_one :"rich_text_#{name}", -> { where(name: name, locale: I18n.locale) },
+                class_name: "ActionText::RichText", as: :record, inverse_of: :record, autosave: true, dependent: :destroy
+        translates name, backend: :action_text
+      end
```

Comment:
Yes this definitely does not belong here. I think the end goal should be to create a separate gem for this, since it's fairly specific.

However, we can make the changes in Mobility necessary to _allow_ that separate gem to work, e.g. making key/value columns configurable, etc.